collection-scripts/gather_monitoring

```diff
@@ -98,17 +108,22 @@ alertmanager_get() {
 monitoring_gather(){
   init
 
+  echo "INFO: Found ${#PROM_PODS[@]} replicas - ${PROM_PODS[*]}"
+
   # begin gathering
   # NOTE || true ignores failures
 
-  prom_get rules rules   || true
-  prom_get alertmanagers alertmanagers  || true
-  prom_get status/config status/config || true
-  prom_get status/flags status/flags || true
-  prom_get status/runtimeinfo status/runtimeinfo || true
-  prom_get_from_replicas status/tsdb status/tsdb || true
+  prom_get alertmanagers   || true
+  prom_get rules    || true
+  prom_get status/config  || true
+  prom_get status/flags  || true
+  prom_get status/runtimeinfo  || true
+
+  # using prom_get_from_replica as the state differs for each replica
+  prom_get_from_replicas targets?state=active active-targets  || true
```

Comment:
nit: `?` is shell-sensitive, so we should probably quote `'targets?state=active'` here.  Although the likelihood of there being a file to match this path expansion is very low, so not a big risk.

Reply:
Good catch :)